### PR TITLE
Create bug_report.md

### PR DESCRIPTION
**Pull request by piotr-dziubecki**

### CODE REVIEW

**Review by hoffmannjan — APPROVED** [review]

**Review by hoffmannjan — APPROVED** [review]

**Comment by hoffmannjan** [comment]
@TomVasile not sure why it's hanging on CI and can't merge it

**Comment by TomVasile** [comment]
@hoffmannjan It's because we ignore changes to markdown files in the workflow and thats all that was changed here. I can merge this as an admin if it's ready to merge.

Example of why this happened:
 https://github.com/casper-ecosystem/casper-js-sdk/blob/master/.github/workflows/ci-casper-client-sdk.yml#L4-L15

**Comment by hoffmannjan** [comment]
@TomVasile thanks for the explanation. If you could merge it to master manually that would be cool :) Thanks!

**Pull request merged**
